Project template (.md) for post-workshop learners (#22)
skadauke/intro-to-r-for-clinicians-chop: exercises/r_project_template.md

---
title: "CHOP R 101"
author: "R User Group + Arcus Ed"

> date: "12/02/2020"

Reviewer: ```suggestion
date: "`r Sys.Date()`"
```

output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
library(tidyverse)
```

> ## Use this script!

Reviewer: ```suggestion
## Use this template!
```

Coders often use templates and code snippets as a starting point for their analysis or find solutions to common problems.

Things we will commonly want to do in an RMarkdown:

* Introduction: what are you trying to accomplish in this script?  
* Import data: bring in relevant data from a .csv, from an API, etc.
* Tidy and transform data
* Exploratory cycle: summarize and visualize your data, check your assumptions
* Share your results: conclusions, findings, and/or export data

Note that this template provides a starting point, but not an unchangeable structure -- feel free to modify, expand, and remove as you see fit (but try to keep the basic mix of code and narrative if possible!)

## Introduction

This is your project! What problem are you trying to solve?

## Import data

> This step usually requires a particular R library to bring in data, such as `readr`, which is happy grabbing data from a variety of sources.

Reviewer: This comment is a little confusing here since you loaded `tidyverse` above and you're not actually loading `readr` here. Suggest: add some language in the `setup` chunk (maybe as `# comments`) to direct people that they might want to load packages here that they will use later on.

Author: Yes good call! I can clarify in the setup chunk that `library(tidyverse)` is actually loading a number of different packages. Will add that lg.

> For example, we might want to bring in a .csv from a location online AND store it in a variable we can use:

Reviewer: Nitpick: throughout the course we've used the terms "capture" and "named object" instead of "store" and "variable" --

```suggestion
For example, we might want to bring in a .csv from a location online AND capture it in a named object so we can use it later:
```
